docs/source/GetStartedGuide.rst

```diff
@@ -264,12 +267,12 @@ In order to do that, you can use ``avocado --show test run ...`` or
     ...
     Job ID: f9ea1742134e5352dec82335af584d1f151d4b85
 
-    START examples/tests/sleeptest.py
+    START 1-sleeptest.py:SleepTest.test
```

Comment:
Three lines above (line 266) there's a bad reference to `examples/tests/sleeptest`, missing the `.py` suffix.